README.md

```diff
@@ -14,52 +14,164 @@ Backend developed by the Digital R&D team for hybrid search apps. Related repos:
+    },
+    {
+        "meta": {"title": "Statue of Zeus at Olympia"},
+        "content": "The Statue of Zeus at Olympia was a giant seated figure, about 12.4 m (41 ft) tall, made by the Greek sculptor Phidias around 435 BC at the sanctuary of Olympia, Greece, and erected in the Temple of Zeus there. Zeus is the sky and thunder god in ancient Greek religion, who rules as king of the gods on Mount Olympus.",
+    },
+    {
+        "meta": {"title": "Temple of Artemis"},
+        "content": "The Temple of Artemis or Artemision (Greek: Ἀρτεμίσιον; Turkish: Artemis Tapınağı), also known as the Temple of Diana, was a Greek temple dedicated to an ancient, localised form of the goddess Artemis (equated with the Roman goddess Diana). It was located in Ephesus (near the modern town of Selçuk in present-day Turkey). By AD 401 it is belived it had been ruined or destroyed.[1] Only foundations and fragments of the last temple remain at the site. ",
+    },
+    {
+        "meta": {"title": "Mausoleum at Halicarnassus"},
+        "content": "The Mausoleum at Halicarnassus or Tomb of Mausolus[a] (Ancient Greek: Μαυσωλεῖον τῆς Ἁλικαρνασσοῦ; Turkish: Halikarnas Mozolesi) was a tomb built between 353 and 351 BC in Halicarnassus (present Bodrum, Turkey) for Mausolus, an Anatolian from Caria and a satrap in the Achaemenid Persian Empire, and his sister-wife Artemisia II of Caria. The structure was designed by the Greek architects Satyros and Pythius of Priene. Its elevated tomb structure is derived from the tombs of neighbouring Lycia, a territory Mausolus had invaded and annexed c. 360 BC, such as the Nereid Monument.",
+    },
+    {
+        "meta": {"title": "Colossus of Rhodes"},
+        "content": "The Colossus of Rhodes (Ancient Greek: ὁ Κολοσσὸς Ῥόδιος, romanized: ho Kolossòs Rhódios; Modern Greek: Κολοσσός της Ρόδου, romanized: Kolossós tis Ródou) was a statue of the Greek sun god Helios, erected in the city of Rhodes, on the Greek island of the same name, by Chares of Lindos in 280 BC. One of the Seven Wonders of the Ancient World, it was constructed to celebrate the successful defence of Rhodes city against an attack by Demetrius I of Macedon, who had besieged it for a year with a large army and navy.",
+    },
+    {
+        "meta": {"title": "Lighthouse of Alexandria"},
+        "content": "The Lighthouse of Alexandria, sometimes called the Pharos of Alexandria (/ˈfɛərɒs/ FAIR-oss; Ancient Greek: ὁ Φάρος τῆς Ἀλεξανδρείας, romanized: ho Pháros tês Alexandreías, contemporary Koine Greek pronunciation: [ho pʰáros tɛ̂ːs aleksandrěːaːs]; Arabic: فنار الإسكندرية), was a lighthouse built by the Ptolemaic Kingdom of Ancient Egypt, during the reign of Ptolemy II Philadelphus (280–247 BC). It has been estimated to have been at least 100 metres (330 ft) in overall height. One of the Seven Wonders of the Ancient World, for many centuries it was one of the tallest man-made structures in the world.",
+    },
+]
```

Comment:
Agree it's really useful to have some data to work with. Only issue is that this appears to be Wikipedia data, which has a permissive license but is distinct to that of our package. Couple of things for your consideration:

* Include a [LICENSE.note](https://github.com/datasciencecampus/transport-network-performance/blob/dev/LICENSE.note) file in the root of the directory. I believe these are [the details of the wikipedia data license](https://en.wikipedia.org/wiki/Wikipedia:Text_of_the_Creative_Commons_Attribution-ShareAlike_4.0_International_License)?
* Rather than storing the data in the README or notebook, it would be better off in a txt file or .py file, whatever is convenient. I would suggest storing this in tests/data/wiki-extract.txt / py or search_backend/data/wiki-extract.txt  / py, depending on whether you expect it to be useful to only package contributors or to users of the package also. If it's in the search_backend directory, people who pip install will also gain access to it.

Reply:
Once that is done, you could update the README & notebook to raise awareness of the stored data and demonstrate how to import it.

Reply:
Good point, have added a commit to address. Have put data in the tests/ folder - I don't have particularly strong opinions either way but I was thinking of it more as an illustration of code usage. It's small enough that you can see what's going on, but otherwise probably a bit limited.

Reply:
@chloepugh brilliant, I'll take a look. This would be really useful for end to end testing purposes also - this data can be used as a fixture for tests. 

I wondered if I could refactor this code to optionally use chromadb instead. It'd be good to have a chat with you about that and whether it would be useful or not.

Reply:
Yes I was thinking previously about allowing it to use different document stores. The reason I didn't carry on with that is annoyingly there's no generic `DocumentStore` type class in Haystack, so where you have things like [this](https://github.com/ministryofjustice/rd-search-backend/blob/1a7526aca87f012ed790b083b8150e08ce6b89d8/search_backend/indexing_pipeline.py#L22) you have to define every type of DocumentStore you want it to be able to use (along with using the corresponding retriever components).

I'd like to get it working with Haystack's `InMemoryDocumentStore`, as that's dead easy to set up and works with both BM25 and semantic search, so is good for testing (just not suitable where you need a persistent document store).

Will add as an issue!